sites/main-site/src/content/hackathon-projects/hackathon-may-2025/cellpainting.md

```diff
@@ -0,0 +1,28 @@
+---
+title: nf-core/cellpainting kickoff
+category: pipelines
+slack: https://nfcore.slack.com/archives/C08H9MWG06T
+image: "/assets/images/events/2025/hackathon-may/cell-painting.jpg"
+image_alt: meme comparing traditional bioimage tools with OME tools for large scale processing
+leaders:
+  kenibrewer:
+    name: Ken Brewer
+    slack: https://nfcore.slack.com/team/U03MWA6LMV1
+  maxulysse:
+    name: Maxime Garcia
+    slack: https://nfcore.slack.com/team/UE6D8290F
+---
+
+This project focuses on creating an nf-core pipeline for processing data for cell painting analysis using CellProfiler, cytotable, pycytominer, and other tools.
+
+## Goals
+
+- Fill out the basic documentation for the planned pipeline in the nf-core/cellpainting repo.
```

Comment:
please label the experience levels of the different goals.

Reply:
Added!